torchqdynamics/solver_utils.py

```diff
@@ -45,3 +45,14 @@ def bexpect(operators: Tensor, state: Tensor) -> Tensor:
     # TODO: Once QTensor is implemented, check if state is a density matrix or ket.
     # For now, we assume it is a density matrix.
     return torch.einsum('bij,...ji->...b', operators, state)
+
+
+def lindbladian(rho: torch.Tensor, H: torch.Tensor, L: torch.Tensor) -> torch.Tensor:
```

Comment:
For consistency with the rest, maybe we could type with `Tensor` instead of `torch.Tensor`. Unless you prefer the other one, I'm fine with both.